data/json/recipes/recipe_deconstruction.json

```diff
@@ -2448,6 +2449,22 @@
     ],
     "flags": [ "BLIND_EASY" ]
   },
+  {
+    "result": "1st_aid",
+    "type": "uncraft",
+    "time": 100,
+    "components": [
+      [ [ "medical_tape", 20 ] ],
+      [ [ "aspirin", 10 ] ],
+      [ [ "bottle_plastic_small", 2 ] ],
+      [ [ "disinfectant", 10 ] ],
+      [ [ "saline", 5 ] ],
```

Comment:
`saline` and `disinfectant` are not spawned in a bottle.

Reply:
There are 2 small bottles for them. AFAIK it is not possible to deconstruct an item into "bottle of liquid", only into bottle and liquid separately.